stable/prometheus-operator/ci/test-values.yaml

```diff
@@ -1658,3 +1678,60 @@ prometheus:
           ## Server name used to verify host name
           ##
           # serverName: ""
+
+  additionalPodMonitors: []
+  ## Name of the PodMonitor to create
+  ##
+  # - name: ""
+
+    ## Additional labels to set used for the PodMonitorSelector. Together with standard labels from
+    ## the chart
+    ##
+    # additionalLabels: {}
+
+    ## Pod label for use in assembling a job name of the form <label value>-<port>
+    ## If no label is specified, the pod endpoint name is used.
+    ##
+    # jobLabel: ""
+
+    ## labels to transfer from the kubernetes pods to the target
+    ##
+    # podTargetLabels: ""
+
+    ## Label selector for pods to which this PodMonitor applies
+    ##
+    # selector: {}
+
+    ## Namespaces from which pods are selected
+    ##
+    # namespaceSelector:
+      ## Match any namespace
+      ##
+      # any: false
+
+      ## Explicit list of namespace names to select
+      ##
+      # matchNames: []
+
+    ## Endpoints of the selected pods to be monitored
+    ##
+    # podMetricsEndpoints: []
+      ## Name of the port this endpoint refers to
+      ## Mutually exclusive with targetPort
+      # - port: ""
+
+      ## Name or number of the target port of the endpoint
+      ## Mutually exclusive with port
+      # - targetPort: ""
+
+      ## Interval at which metrics should be scraped
+      ##
+      #   interval: 30s
+
+      ## HTTP path to scrape for metrics
+      ##
+      #   path: /metrics
+
+      ## HTTP scheme to use for scraping
+      ##
+      #   scheme: http
```

Comment:
Might be useful to include a link to the api documentation for this section: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#podmetricsendpoint

Reply:
Added a link above